README.md

```diff
@@ -152,15 +146,65 @@ Use <kbd>Ctrl + Shift + F</kbd> to see auto-detected subtitle tracks for streami
 
 <kbd>Ctrl + Shift + F</kbd> also allows you to load an empty subtitle track so that you can extract audio and screenshots from streaming video without loading subtitles.
 
-### Extension side panel UI
+#### Filtering subtitle text
+
+If you'd like to filter out specific instances subtitle text, one way to do so is by using a regular expression (regex). asbplayer can match any sequence following a specified regex pattern and remove the matches.
+
+Under the MISC section in asbplayer settings, locate the "Subtitle regex filter" textbox. Enter an appropriate regex to filter desired content.
+You can replace filtered content similarly by entering a string into the "Subtitle regex filter text replacement" textbox. Leaving this blank will simply remove the content.
+
+Potential use cases for filtering include:
+ - Remove names enclosed by parenthesis to indicate speakers  (i.e. "**（山田）**　元気ですか？")
+ - Remove indications enclosed by brackets that sound or music that is playing (i.e. "**\[PLAYFUL MUSIC]**")
+
+Helpful materials on writing regular expressions:
+- [GeeksForGeeks regex tutorial](https://www.geeksforgeeks.org/write-regular-expressions/)

```

Comment:
Let's limit the resources we provide to a single high-quality one that doesn't have ads. Most people who know nothing about regexes will likely not want to click through multiple links to just learn about regexes. I suggest this one: https://regexlearn.com/playground

Reply:
this is a wonderful resource!!